### ISSUE TITLE
Make page GDPR compliant

### ISSUE THREAD

**Issue by tklebel** [tklebel/thomasklebel#5]
# Dear visitors from the Google font issue

This is just the source for my private homepage. I should not have cross-referenced the issue about Google fonts, I did it accidentally. Sorry!

The points below are what I initially planned to make my site somewhat compliant with GDPR, but I ended up doing things differently. The list is thus not up to date.

-----

Hints:
- https://discourse.gohugo.io/t/hugo-vs-the-general-data-protection-regulations-gdpr-in-eu-eea/11526
- https://drschwenke.de/google-analytics-datenschutz-muster-faq/

Issues: Any form of data forwarding without consent

- [x] GoogleAnalytics cookies should be only activated after consenting
- [x] Disqus should be disabled by default
- [x] Social buttons should not transmit data
- [x] There should be a page about data privacy and cookie policy which should explain hosting and all other issues https://www.netlify.com/gdpr/
- [x] I should have a contract with Google that regulates how they use my data from google analytics. I have this kind of contract, it is specified here: https://support.google.com/analytics/answer/3379636 and an interesting article here: https://drschwenke.de/google-analytics-datenschutz-zusatz-zur-datenverarbeitung/
- [ ] google fonts seem still to be problematic: `https://github.com/google/fonts/issues/1495`

Wait for Hugo v0.41.0, which should resolve many issues: https://github.com/gohugoio/hugo/releases

**Comment by tklebel** [comment]
Somehow with the new Hugo version Disqus comments currently do not work at all.

**Comment by tklebel** [comment]
Google Analytics tracking seems to be gone too

**Comment by davelab6** [comment]
Google Fonts acts as a "data controller" for any personal data that Google processes in connection with your use of Google Fonts web and Android APIs.

**Comment by tklebel** [comment]
@davelab6 Thank you and sorry: I didn't mean to reference my issue in the thread, just wanted to bookmark it. Next time I will put it as code to avoid the reference.

**Comment by ozdemirburak** [comment]
Hey Thomas, I came here from the Google fonts issue, just writing here to ask why you disabled Disqus.

I don't think you need to disable Disqus by default, just informing your users about you are using Disqus would seem ok as they are the controller and also follow DNT as this article mentions that they are handling the issue, unless a publisher integrates Disqus with their own user management.

https://blog.disqus.com/update-on-privacy-and-gdpr-compliance

And there is also this comment answered by a Disqus staff in that thread.

https://blog.disqus.com/update-on-privacy-and-gdpr-compliance#comment-3912683006

And for the Google Analytics, you probably need to anonymize IP addresses or ask user's permission.

https://developers.google.com/analytics/devguides/collection/analyticsjs/ip-anonymization

**Comment by tklebel** [comment]
Hi @ozdemirburak,
I read the discussion about Disqus and actually found the comment too. So what I settled on was to state that I am using Disqus in my [privacy declaration](https://thomasklebel.eu/privacy/), and offer a direct link to the opt-out at Disqus.

As for Google Analytics, I am anonymizing IP addresses (an option for privacy settings was added in the Hugo-academic theme), and I further implemented an opt-out which should prevent tracking on my site (although I am not 100% sure about the JS involved, since I am no web developer).

I didn't update the initial post but simply closed the issue...
